app.py

```diff
@@ -1,79 +1,114 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
-from item import TrelloItem
+from item import Item
 from trello import TrelloClient
 from viewModel import ViewModel
 import os
+import pymongo
+from datetime import datetime
+from bson.objectid import ObjectId
+
+date_time_format = "%Y-%m-%dT%H:%M:%S.%fZ"
+
 
 def get_trello_board(api_key, token, board_id):
-        client = TrelloClient(
-            api_key=api_key,
-            token=token
-        )
-        
-        return client.get_board(board_id)
-        
-    
-def get_items(api_key, token, board_id):
+    client = TrelloClient(
+        api_key=api_key,
+        token=token
+    )
+
+    return client.get_board(board_id)
+
+
+def get_items(collection):
     items = []
-    board = get_trello_board(api_key, token, board_id)
-    board_lists = board.list_lists()
-    for list_item in board_lists:
-        for card in list_item.list_cards():
-            items.append(TrelloItem(card.id, list_item.name,card.name, card.dateLastActivity))
-            
+
+    for item in collection.find():
+        items.append(
+            Item(
+                item['_id'],
+                item['status'],
+                item['title'],
+                item['last_modified']
+            )
+        )
+
     return items
 
+
+def add_new_item(collection, title):
+    collection.insert_one(
+        {
+            "title": title,
+            "status": 'ToDo',
+            "last_modified": datetime.now()
+        }
+    )
+
+
+def mark_item_as_complete(collection, id):
+    collection.update_one(
+        {"_id": ObjectId(id)},
+        {
+            "$set": {
+                "status": 'Done',
+                "last_modified": datetime.now()
+            }
+        }
+    )
+
+
+def mark_item_as_uncomplete(collection, id):
+    collection.update_one(
+        {"_id": ObjectId(id)},
+        {
+            "$set": {
+                "status": 'ToDo',
+                "last_modified": datetime.now()
+            }
+        }
+    )
+
+
+def get_db_collection():
+    dbClientUri = f"mongodb+srv://{os.getenv('MONGO_DB_USER_NAME')}:{os.getenv('MONGO_DB_PASSWORD')}@cluster0.59kpk.mongodb.net/?retryWrites=true&w=majority"
```

Comment:
We should extract out the host address (`cluster0.59kpk.mongodb.net`) here (and in your e2e test) into an environment variable, so that alternative mongodb clusters could also be used